plugins/examples/meta-plugin/build.gradle

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// A meta plugin packaging example that bundles two plugins in a single zip.
+
+apply plugin: 'elasticsearch.standalone-rest-test'
+apply plugin: 'elasticsearch.rest-test'
+
+File plugins = new File(buildDir, 'plugins-unzip')
+
+// unzip the rescore plugin in a tmp directory
+task unzipRescorePlugins(type:Copy, dependsOn:':example-plugins:rescore:bundlePlugin') {
+    File pluginsRescore = new File(plugins, 'example-rescore')
+    from { zipTree(project(':example-plugins:rescore').bundlePlugin.outputs.files.singleFile) }
```

Comment:
Can you create 2 dummy plugins inside this example (as sub projects within the source here) instead of trying to use other existing examples? What is here now will cause the examples to not be installable at the same time, and we don't need real functionality out of the bundled plugins, just an example that it is possible and they are loaded.